about.html

```diff
@@ -0,0 +1,269 @@
+          aspirational. Our 150+ expert speakers bring fresh new insights from
+          the tech industry that can help transform the world of tomorrow.
+        </p>
+        <div class="card m-5 mt-2 border-3 border-white bg-light">
+          <div class="card-body mx-auto">
+            <img
+              src="img/The_Next_Web_Logo.png"
+              alt="The_Next_Web_Logo"
+              class="logo"
+            />
+          </div>
+        </div>
+      </section>
+      <hr />
+      <section id="previous-conference">
+        <p class="mx-4 mt-4 display-6 secondary-color text-center">
+          See the past, TNW conference, NOW.
+        </p>
+        <div class="short-line mx-auto"></div>
+        <p class="mx-4 mb-5 text-black-50 text-center">
+          Take a look at the last two TNW Global Conference which took place
+          always In • Amsterdam, June 16 & 17.
+        </p>
+        <ul class="conferences-list m-4 mb-5">
+          <li>
+            <img
+              src="img/conferences/conference2020.jpg"
+              alt="conferece 2020 image"
+              class="img-thumbnail rounded mx-auto d-block"
+            />
+            <div class="centered">
+              <h5>
+                The Next Web conference <br />
+                <br />
+                2020
+              </h5>
+            </div>
+            <div class="bg-blur"></div>
+          </li>
+          <li>
+            <img
+              src="img/conferences/conference2018.jpg"
+              alt="conferece 2018 image"
+              class="img-thumbnail rounded mx-auto d-block"
+            />
+            <div class="centered">
+              <h5>
+                The Next Web conference <br /><br />
+                2018
+              </h5>
+            </div>
+            <div class="bg-blur"></div>
+          </li>
+        </ul>
+      </section>
+      <section
+        id="partners"
+        class="partners d-flex flex-column justify-content-center align-items-center only-mobile"
+      >
+        <h3 class="text-center mt-3">Partners</h3>
+        <div class="short-line mx-auto mt-3"></div>
+        <div class="partners-logos mt-3">
+          <img src="img/partners/mozilla.svg" alt="mozilla icon" />
+          <img src="img/partners/google.svg" alt="google icon" />
+          <img src="img/partners/naver.svg" alt="naver icon" />
+          <img src="img/partners/tokopedia.png" alt="tokopedia icon" />
+          <img src="img/partners/airbnb.svg" alt="airbnb icon" />
+        </div>
+      </section>
+    </main>
+    <footer class="d-flex justify-content-center align-items-center py-4 gap-4">
+      <div><img src="img/TNW_Logo.png" alt="TNW logo" /></div>
+      <div>
+        <p>2022 TNW: The Next Web Conference.</p>
+        <p class="only-desktop">
+          Copyright © 2006—2022, The Next Web B.V. Made with ❤ in Amsterdam.
+        </p>
+        <p>Some Rights Reserved.</p>
+      </div>
+    </footer>
```

Comment:
- [ ] [REQUIRED] Great implementation here, however, let's ensure that the height for the footer and the partner's section are the same.
### Your Implementation
![Screenshot from 2022-04-22 23-51-23](https://user-images.githubusercontent.com/58692590/164815238-6a7f30a0-d121-4090-998a-002db34af36e.png)
### Behance Design
![Screenshot from 2022-04-22 23-51-31](https://user-images.githubusercontent.com/58692590/164815274-d8dd468a-4d95-4a6a-aac3-d9ad17b24f08.png)

Reply:
This Implementation is for the desktop version of the main page,